nemo_curator/scripts/semdedup/config.yaml

```diff
@@ -0,0 +1,35 @@
+cache_dir: '/raid/vjawa/semdedup_debug'
```

Comment:
Can you change the defaults to something more generic?

Reply:
Done